libpod/options.go

```diff
@@ -1208,14 +1206,14 @@ func WithDNSSearch(searchDomains []string) CtrCreateOption {
 }
 
 // WithDNS sets additional name servers for the container.
-func WithDNS(dnsServers []string) CtrCreateOption {
+func WithDNS(dnsServers []net.IP) CtrCreateOption {
```

Comment:
Why this change?

Reply:
not sure, I think that was a while back when I was first trying to make the codepath work. I just changed it back to []string and same for the inspect option